lib/data/property_data.dart

```diff
@@ -330,3 +334,14 @@ enum Feature {
     }
   }
 }
+
+class User extends Equatable {
```

Comment:
@suesitran Because you didn't mention about the path of User class in code plan, so should i move it ? And if i need to move, then should i move to `data.dart` class (Where you have specified UserDetail class)  or move it to a completely new file ?

Reply:
Please reuse UserDetail, add param for avatar if needed